orca/orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/tasks/MonitorWebhookTask.java

```diff
@@ -172,7 +172,7 @@ public TaskResult execute(StageExecution stage) {
     }
 
     monitor.setBody(response.getBody());
-    monitor.setStatusCode(response.getStatusCode());
+    monitor.setStatusCode(HttpStatus.valueOf(response.getStatusCode().value()));
```

Comment:
```suggestion
monitor.setStatusCode(response.getStatusCode());
```
works for me locally.